src/deploy.go

```diff
@@ -3,35 +3,27 @@ package main
 import (
 	"encoding/json"
 	"errors"
+	"fmt"
 	"os/exec"
 	"path"
 	"strings"
 
 	log "github.com/sirupsen/logrus"
 )
 
-func deployGoRoutine(callbackID string,
-	submissionDataMap map[string]interface{}) {
-	if chatPostMessage(submissionDataMap["channel"].(string),
-		"Deployment in Progress", nil) == false {
-		log.Warn("Some error occured")
+func deployGoRoutine(callbackID string, submissionDataMap map[string]interface{}) {
+	if err := chatPostMessage(submissionDataMap["channel"].(string), "Deployment in Progress", nil); err != nil {
+		log.Errorf("cannot post begin deployment chat message - %v", err)
 		return
 	}
+	log.Infof("beginning deployment of %s on server %s with subdomain as %s with callback_id as %s", submissionDataMap["git_repo"], submissionDataMap["server_name"], submissionDataMap["subdomain"], callbackID)
```

Comment:
line too long. why not wrap it?

Reply:
I had been using gofmt to format all the code, I write. That is the standard way to format go code. 
The official doc says - https://golang.org/doc/effective_go.html#formatting 
`Go has no line length limit. Don't worry about overflowing a punched card. If a line feels too long, wrap it and indent with an extra tab.`
I will not stick to a hard 80 char limit, but format lines, which are too big later.